### PR TITLE
activatePreviousPane

### PR DESCRIPTION
**Pull request by aminya**
Fixes https://github.com/JunoLab/atom-indent-detective/issues/21

### CODE REVIEW

File: src/selector.ts [hunk 1]
```diff
@@ -56,6 +56,14 @@ export class Selector {
         this.indentListView.focus()
     }
 
+    // Hide a selector
+    hide() {
+        // focus on the previous active pane
+        atom.workspace.activatePreviousPane()
```
Comment (aviatesk) [inline src/selector.ts:62]: I don't think this isn't guaranteed to be really "previous" pane.
https://github.com/atom/atom/blob/v1.45.0/src/workspace.js#L1654

Comment (aminya) [reply src/selector.ts]: fixed in https://github.com/JunoLab/atom-indent-detective/pull/22/commits/24759cf119c33bd9856a04a65cca7c63c644f0bf

File: src/selector.ts [hunk 2]
```diff
@@ -50,12 +51,22 @@ export class Selector {
...
         this.modalPanel.show()
         this.indentListView.focus()
     }
 
+    // Hide a selector
+    hide() {
+        // focus on the previous active pane
+        this.previousActivePane.activate()
+        // hide modal panel
+        this.modalPanel.hide()
```
Comment (aviatesk) [inline src/selector.ts:67]: let's call `hide` first, that can mess a focus iirc

Comment (aminya) [reply src/selector.ts:65]: Fixed in https://github.com/JunoLab/atom-indent-detective/pull/22/commits/3a4b38f933ced625144fbd755ab6a372f421d16d

**Comment by aviatesk** [comment]
nice

**Pull request merged**